MANIFEST.in

```diff
@@ -4,5 +4,6 @@ recursive-include autosklearn/metalearning/files *.txt
 include autosklearn/util/logging.yaml
 recursive-include autosklearn *.pyx
```

Comment:
This is legacy right? maybe we should add another issue to clean this one up

Reply:
Yes, indeed. Good point.

Reply:
https://github.com/automl/auto-sklearn/pull/934